src/features/hotspots/details/HexBadge.tsx

```diff
@@ -54,13 +63,13 @@ const HexBadge = ({
   }, [hotspotId, t])
 
   const color = useMemo(() => {
-    if (!rewardScale) return 'white'
+    if (!rewardScale) return 'grayText'
 
     return generateRewardScaleColor(rewardScale)
   }, [rewardScale])
 
   const scaleString = useMemo(() => {
-    if (!rewardScale) return ''
+    if (!rewardScale) return `0${decimalSeparator}00`
 
     if (rewardScale === 1) return `1${decimalSeparator}00`
```

Comment:
Is `if (rewardScale === 1) return `1${decimalSeparator}00` still needed?

Reply:
Yeah it's needed to format the text properly when the scale is 1.